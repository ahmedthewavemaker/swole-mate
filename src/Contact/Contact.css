.contact{
    background-color: rgba(0,0,0,0.8);
   
}

p{
    font-size: 30px;
    font-weight: bolder;
    color: gold;
}