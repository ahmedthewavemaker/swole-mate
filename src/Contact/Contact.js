import React from 'react';
import './Contact.css'


function Contact() {
    return (
      <div className='contact'>
        <p>Ahmed E</p>
        <p>Number: (132)-222-3334</p>
        <p>Email: elnassehag@alumni.vcu.edu</p>
      </div>
    );
  }
  export default Contact;