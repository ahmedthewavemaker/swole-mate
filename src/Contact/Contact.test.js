import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Contact from './Contact';
import { MemoryRouter } from 'react-router-dom';

describe('Contact component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<MemoryRouter> <Contact />
      </MemoryRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<MemoryRouter> <Contact />
        </MemoryRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });


});
