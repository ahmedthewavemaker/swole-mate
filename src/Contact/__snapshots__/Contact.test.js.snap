// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders the UI as expected 1`] = `
Array [
  " ",
  <div
    className="contact"
  >
    <p>
      Ahmed E
    </p>
    <p>
      number: (132)-222-3334
    </p>
    <p>
      email: mywholeemail@email.com
    </p>
  </div>,
]
`;
